src/node/interfaces.cpp

```diff
@@ -112,6 +112,38 @@ class NodeImpl : public Node
         }
     }
     bool shutdownRequested() override { return ShutdownRequested(); }
+    bool isSettingIgnored(const std::string& name) override
+    {
+        bool ignored = false;
+        gArgs.LockSettings([&](util::Settings& settings) {
+            if (auto* options = util::FindKey(settings.command_line_options, name)) {
+                ignored = !options->empty();
+            }
+        });
+        return ignored;
+    }
+    util::SettingsValue getPersistentSetting(const std::string& name) override { return gArgs.GetPersistentSetting(name); }
+    void updateSetting(const std::string& name, const util::SettingsValue& value) override
+    {
+        gArgs.LockSettings([&](util::Settings& settings) {
```

Comment:
```cpp
updateRwSetting(name, value, /* write = */ true)
```

Reply:
re: https://github.com/bitcoin/bitcoin/pull/15936#discussion_r878685831

> ```c++
> updateRwSetting(name, value, /* write = */ true)
> ```

Thanks renamed to `updateRwSetting` to be consistent with the Chain interface